packages/client/src/generation/generateClient.ts

```diff
@@ -287,20 +287,18 @@ export async function generateClient({
     await copyFile(datamodelPath, datamodelTargetPath)
   }
 
-  if (!generator?.isCustomOutput) {
-    const packageJsonTargetPath = path.join(finalOutputDir, 'package.json')
-    const pkgJson = JSON.stringify(
-      {
-        name: '.prisma/client',
-        main: 'index.js',
-        types: 'index.d.ts',
-        browser: 'index-browser.js',
-      },
-      null,
-      2,
-    )
-    await writeFile(packageJsonTargetPath, pkgJson)
-  }
+  const packageJsonTargetPath = path.join(finalOutputDir, 'package.json')
+  const pkgJson = JSON.stringify(
+    {
+      name: buildGeneratedPackageJsonName(finalOutputDir),
```

Comment:
Potentially stupid question: Does the name of the package matter?

Reply:
I don't think it does, actually, unless users want to publish generated package, in which case they probably want to have their own name anyway. After thinking about it, we could probably just always leave `.prisma/client` there or even remove it completely.